zendesk/zd_ticket_metrics_service.go

```diff
@@ -0,0 +1,53 @@
+package zendesk
+
+import (
+	"fmt"
+	"time"
+)
+
+// Ticket represents a Zendesk Ticket.
+//
+// Zendesk Core API docs: https://developer.zendesk.com/rest_api/docs/core/ticket_metrics
+
+type TicketMetric struct {
```

Comment:
Good work on including the link but first line of comment is misleading. Ticket or TicketMetrics?

Reply:
Updated.